src/log.py

```diff
@@ -18,16 +19,13 @@ def rotateLogs(self):
 			with gzip.open("logs/wrapper/oldWrapper.log.gz", "w") as f:
 				f.write(originalLog)
 			os.remove("wrapper.log")
-		if self.file:
-			self.file.close()
-			time.sleep(1) # Possibly fix Windows 'in-use' error. Ugh.
 		if os.path.exists("logs/wrapper/current.log"):
 			with open("logs/wrapper/current.log", "r") as f:
 				logData = f.read()
-			with gzip.open("logs/wrapper/%s.log.gz" % time.strftime("%Y-%m-%d_%H-%M-%S"), "w") as f:
-				f.write(logData)
-			os.remove("logs/wrapper/current.log")
-		self.file = open("logs/wrapper/current.log", "a")
+		os.remove("logs/wrapper/current.log")
```

Comment:
You put os.remove outside of the if os.path.exists - if current.log doesn't exist, this will fail.